components/Input/index.jsx

```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+
+const Input = ({ id, className, ...attributes }) => (
+  <input
+    className={`w-full h-full bg-dark text-white text-base-large rounded-xl px-3 pt-[19px] pb-[15px] focus:outline-none focus:shadow-none ${className}`}
```

Comment:
Maybe moving these default class names to a `const` make this a bit cleaner 💡 

```suggestion
  const DEFAULT_CLASS_NAMES = 'w-full h-full bg-dark text-white text-base-large rounded-xl px-3 pt-[19px] pb-[15px] focus:outline-none focus:shadow-none';

    className={`${DEFAULT_CLASS_NAMES} ${className}`}
```

Reply:
done in https://github.com/tyrro/react-survey/commit/dbf9d1e261944188cc87b0d6f495a313bea9b890